networks/suzuka/suzuka-full-node/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+pub mod partial;
+
+#[cfg(test)]
+pub mod tests;
+
+pub trait MonzaFullNode {
```

Comment:
```suggestion
pub trait SuzukaFullNode {
```

Reply:
Sorry I pushed this early without going into the code to rename these things.
I've done a search-and-replace on such names now.